### PR TITLE
fix(scrollbar): fix style of scrollbar

### LINKED ISSUE #964: Bug & improve with scrollbar
1. In many components like `select`, `time-select`, once click the scrollbar, when mouse up, it still scroll.
2. image
image
image
Scrollbar should has default `height: 100%`, even though, the horizontal scrollbar has a native bar.

### PR DESCRIPTION
**Pull request by kooriookami**
hide native scrollbar horizontal, give default height: 100%

fix #964

Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by kooriookami** [comment]
https://test.kooriookami.top/deploy-preview-971--element-plus/#/zh-CN

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
cc @SimonaliaChen

File: packages/theme-chalk/src/scrollbar.scss [hunk 1]
```diff
@@ -17,13 +18,8 @@
   @include e(wrap) {
     overflow: scroll;
     height: 100%;
-
-    @include m(hidden-default) {
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/scrollbar.scss:21]: This is only useful when `native = false`, do not remove this rule.

-

**Review by SimonaliaChen — CHANGES_REQUESTED** [review]

**Comment by SimonaliaChen** [comment]
After investigation, I found out that the root cause you mentioned in #964 is that the `margin-bottom` style not working. So I prefer only change https://github.com/element-plus/element-plus/blob/dev/packages/scrollbar/src/index.vue#L8 to `native ? '' : 'el-scrollbar__wrap--hidden-default'` to fix it. Would you like to recorrect it ? or I will submit another pr.

**Comment by kooriookami** [comment]
> After investigation, I found out that the root cause you mentioned in #964 is that the `margin-bottom` style not working. So I prefer only change https://github.com/element-plus/element-plus/blob/dev/packages/scrollbar/src/index.vue#L8 to `native ? '' : 'el-scrollbar__wrap--hidden-default'` to fix it. Would you @kooriookami like to recorrect it ? or I will submit another pr.

Only change this will cause another problems. The bottem was cut off.
image

**Comment by JeremyWuuuuu** [comment]
I have merged the dependency request, you may rebase that then deploy again, so that we can see the result, also, is this only reproducible on a Windows machine? Because the scrollbar on Windows doesn't act as same as a Mac machine.

**Comment by SimonaliaChen** [comment]
> > After investigation, I found out that the root cause you mentioned in #964 is that the `margin-bottom` style not working. So I prefer only change https://github.com/element-plus/element-plus/blob/dev/packages/scrollbar/src/index.vue#L8 to `native ? '' : 'el-scrollbar__wrap--hidden-default'` to fix it. Would you @kooriookami like to recorrect it ? or I will submit another pr.
> 
> Only change this will cause another problems. The bottem was cut off.
> image

I see. `margin-bottom` only works in certain cases. And try this.

```diff
    @include m(hidden-default) {
      scrollbar-width: none;
      &::-webkit-scrollbar {
-       width: 0;
-       height: 0;
+       opacity: 0;
      }
    }
```

**Comment by kooriookami** [comment]
> > > After investigation, I found out that the root cause you mentioned in #964 is that the `margin-bottom` style not working. So I prefer only change https://github.com/element-plus/element-plus/blob/dev/packages/scrollbar/src/index.vue#L8 to `native ? '' : 'el-scrollbar__wrap--hidden-default'` to fix it. Would you @kooriookami like to recorrect it ? or I will submit another pr.
> > 
> > 
> > Only change this will cause another problems. The bottem was cut off.
> > image
> 
> I see. `margin-bottom` only works in certain cases. And try this.
> 
> ```diff
>     @include m(hidden-default) {
>       scrollbar-width: none;
>       &::-webkit-scrollbar {
> -       width: 0;
> -       height: 0;
> +       opacity: 0;
>       }
>     }
> ```

Well, it still weird. The visiable height of the div is 17 px less.
image

**Comment by SimonaliaChen** [comment]
I'm afraid it's the best solution as I can imagine, both elegant and simple. Shall we take that for the temporary fix, pr is still welcome later

**Comment by kooriookami** [comment]
> I'm afraid it's the best solution as far as I can imagine, both elegant and simple. Shall we take that for the temporary fix, pr is still welcome later

I made changes according to what you said.

File: packages/theme-chalk/src/scrollbar.scss [hunk 2]
```diff
@@ -15,14 +16,17 @@
   }
 
   @include e(wrap) {
-    overflow: scroll;
+    overflow: auto;
     height: 100%;
 
+    &::-webkit-scrollbar {
+      display: none;
```
Comment (SimonaliaChen) [inline packages/theme-chalk/src/scrollbar.scss:23]: Like I said before, native scrollbar will display when `native=true`. Do not add this

**Review by SimonaliaChen — CHANGES_REQUESTED** [review]

File: packages/theme-chalk/src/scrollbar.scss (hunk 2 above)
Comment (kooriookami) [reply packages/theme-chalk/src/scrollbar.scss]: OK

**Review by SimonaliaChen — APPROVED** [review]

**Pull request merged**